etc/settings.cfg.sh

```diff
@@ -25,9 +25,7 @@ export EUPSPKG_SOURCE=${EUPSPKG_SOURCE:-git}
 export EUPS_PKGROOT=$LSSTSW/distserver/production
 
 # the location of source repositories
-BASE='git://git.lsstcorp.org/LSST'
-SBASE='https://stash.lsstcorp.org/scm'
-export REPOSITORY_PATTERN="$BASE/sims/%(product)s.git|$BASE/DMS/%(product)s.git|$BASE/DMS/devenv/%(product)s.git|$BASE/DMS/testdata/%(product)s.git|$BASE/external/%(product)s.git|$SBASE/sim/%(product)s.git"
+export REPOFILE=${LSSTSW}/etc/repos.yaml
```

Comment:
Uh oh.  `--repos="${REPOSFILE}"` above; `export REPOFILE` here.
Also, I think it's a good idea to still have a backup REPOSITORY_PATTERN.  Furthermore, it might be good to have the `repos.yaml` only contain things that need to be remapped, rather than everything.